src/Wallabag/CoreBundle/Resources/views/themes/material/Static/howto.html.twig

```diff
@@ -18,7 +18,7 @@
                     <div class="col s12">
                         <h5>{{ 'howto.top_menu.browser_addons'|trans }}</h5>
                         <ul>
-                            <li><a href="https://addons.mozilla.org/firefox/addon/wallabag/" target="_blank">{{ 'howto.browser_addons.firefox'|trans }}</a></li>
+                            <li><a href="https://addons.mozilla.org/fr/firefox/addon/wallabag-v2/" target="_blank">{{ 'howto.browser_addons.firefox'|trans }}</a></li>
```

Comment:
Same comment here: why using the french URL?

https://addons.mozilla.org/firefox/addon/wallabag-v2/